docs/topics/spiders.rst

```diff
@@ -680,6 +680,42 @@ SitemapSpider
 
         Default is ``sitemap_alternate_links`` disabled.
 
+    .. method:: sitemap_filter(entries)
+
+        Specifies a function to filter sitemap entries and their attributes.
+
+        For example::
+
+            <url>
+                <loc>http://example.com/</loc>
+                <lastmod>2005-01-01</lastmod>
+            </url>
+
+        We can define a ``sitemap_filter`` function to filter ``entries`` by date::
+
+            def sitemap_filter(entries):
```

Comment:
Shouldn't it have self as a first argument? What do you think about showing a complete example, with subclassing?

Reply:
Good observation! Fixed.

Also took the chance to improve the docs as suggested by you.